iac-extensions/arm/src/main/java/org/sonar/iac/arm/visitors/ArmSymbolVisitor.java

```diff
@@ -45,6 +49,7 @@
  * Those Symbol/Usage can later be used in checks to  report issues in the variable flow.
  */
 public class ArmSymbolVisitor extends TreeVisitor<InputFileContext> {
+  private static final Pattern ASSIGNED_IDENTITIES_PATTERN = Pattern.compile("(?U)variables\\('(?<variableName>[a-zA-Z_]\\w*)'\\)");
```

Comment:
Nitpick. The `(?U)` doesn't work with [a-zA-Z], maybe it should be replaced by:

```suggestion
  private static final Pattern ASSIGNED_IDENTITIES_PATTERN = Pattern.compile("(?U)variables\\('(?<variableName>[\p{L}_]\\w*)'\\)");
```

if you like to fully support Unicode